pkg/tenant/db/projects.go

```diff
@@ -21,7 +21,7 @@ const (
 
 // Project states to return to the frontend.
 const (
-	ProjectStatusIncomplete = "Incomplete Setup"
+	ProjectStatusIncomplete = "Incomplete"
```

Comment:
Can you check with @eschmier on if this status should be `Incomplete Setup` or `Incomplete`?